components/director/pkg/graphql/schema.graphql

```diff
@@ -108,6 +108,7 @@ enum ConstraintScope {
 enum ConstraintType {
 	PRE
 	POST
```

Comment:
Can you add:
LOAD_FORMATIONS
SELECT_SYSTEMS_FOR_FORMATION 
keys in enum TargetOperation